### PR TITLE
feat: add stream docs

### PR DESCRIPTION
**Pull request by mmeloni**
This PR add docs related to new chain stream api

### CODE REVIEW

**Review by aarmoa — CHANGES_REQUESTED** [review]

File: source/includes/_streamrpc.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+	"fmt"
+	"github.com/InjectiveLabs/injective-core/injective-chain/stream/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+func main() {
+	tlsConfig := tls.Config{InsecureSkipVerify: true}
+	creds := credentials.NewTLS(&tlsConfig)
+	cc, err := grpc.Dial("staging.stream.injective.network:443", grpc.WithTransportCredentials(creds))
```
[the remark this answers is not in the document]
Comment (mmeloni) [reply source/includes/_streamrpc.md:34]: This is the idiomatic go gRPC way to connect to a grpc server. In others example there is the need to use more sofisticated connection solutions in order to interact with the chain. Streamer is a simple grpc server that does not requires to use such complexity. For golang developers this is very easy to read.

[the remark this answers is not in the document]
Comment (mmeloni) [reply source/includes/_streamrpc.md:34]: But I got your point. When we will deploy prod stream dns I will change this part, ty

**Review by aarmoa — APPROVED** [review]
The changes look good to me since we are going to update the example when all other environment URLs are available

**Comment by aarmoa** [comment]
Closing this PR because its changes have been included in https://github.com/InjectiveLabs/docs/pull/139

**Pull request closed without merging**